kroxylicious-filters/kroxylicious-record-encryption/src/main/java/io/kroxylicious/filter/encryption/RecordEncryptionFilter.java

```diff
@@ -139,6 +145,17 @@ private CompletionStage<ProduceRequestData> maybeEncodeProduce(ProduceRequestDat
                 });
     }
 
+    private void generateMetrics(String virtualClusterName, Set<String> unresolvedTopicNames, Map<String, TopicProduceData> topicNameToData) {
```

Comment:
This is how I am currently to counting the record for the topics that are not encrypted and creating individual counters for the topics